packages/manager/apps/web/client/app/hosting/boost/enable/hosting-boost-enable.controller.js

```diff
@@ -13,6 +13,19 @@ angular
         request: false,
       };
 
+      User.getUser()
+        .then((user) => {
+          HostingBoost.getBoostPrice(user.ovhSubsidiary).then((catalog) => {
+            catalog.addons.forEach((addon) => {
+              if (addon.planCode === 'consumption-perf2014') {
```

Comment:
Maybe you can `find` the addon instead of looping
```suggestion
             const addon = catalog.addons.find(({ planCode }) => planCode === 'consumption-perf2014')
```